src/App/Services/MobilePasswordRepromptService.cs

```diff
@@ -34,5 +40,11 @@ public async Task<bool> ShowPasswordPromptAsync()
 
             return await _platformUtilsService.ShowPasswordDialogAsync(AppResources.PasswordConfirmation, AppResources.PasswordConfirmationDesc, validator);
         }
+
+        public async Task<bool> Enabled()
+        {
+            var keyConnectorService = ServiceContainer.Resolve<IKeyConnectorService>("keyConnectorService");
```

Comment:
This is a little bit of a circular dependency problem. `MobilePasswordRepromptService` is actually created in platform specific code (for iOS its `iOSCoreHelpers`) before our `ServiceContainer` is created. The `KeyConnectorService` constructor uses many services created in the `ServiceContainer` so we can't register it sooner. But the `MobilePasswordRepromptService` uses the `KeyConnectorService`.

Anyways, thats all to say we can't put it in the constructor yet since it hasn't been created.

If you can think of a way around that @eliykat let me know.

Reply:
Ok, that makes sense and that seems like a reasonable way to deal with it.